plone-5.2.x.cfg

```diff
@@ -3,4 +3,6 @@ extends =
     base.cfg
     http://dist.plone.org/release/5.2.1/versions.cfg
 find-links += http://dist.plone.org/thirdparty/
-versions=versions
```

Comment:
@rodfersou do you know why it is safe to remove those now? In the first Plone 5.2 buildout did not work without this line.

Reply:
I do not know if it's safe, but for sure it's not doing what we think it should do.